server/livecheck.py

```diff
@@ -87,18 +130,22 @@ def processCodeData(data):
             return {
                 "machine_id": machine_id,
                 "election_id": election_id,
-                "timestamp": timestamp
+                "timestamp": timestamp,
             }
 
         assert vxsuite_version == "v4"
-        system_hash, software_version, machine_id, election_id, timestamp = fields
+        machine_id = cert_details[VX_CUSTOM_CERT_FIELD["MACHINE_ID"]]
```

Comment:
Maybe we should `return None` if we can't find the machine ID in the cert

I know we could try to be backwards-compatible with older alpha/beta versions of v4 and support both machine ID in the cert and in the fields, but that's probably overdoing the backwards compatibility and adding unnecessary complexity

Unless @benadida has some upcoming demos where he's bound to an older v4 image

Reply:
Ah okay, that makes sense. I will update to `return None` and will adjust if Ben needs it